ola.spec

```diff
@@ -1,91 +1,108 @@
-%define name    ola
-%define version 0.3.1
-%define release %mkrel 1
-
-
-Name:      %{name}
-Version:   %{version}
-Release:   %{release}
-Summary:   OLA - Open Lighting Architecture
-Group:     Other
-License:   GPL
-URL:       https://github.com/OpenLightingProject/ola
-# Other doc found at:
-#  http://www.opendmx.net/index.php/LLA_0.3
-#  http://www.opendmx.net/index.php/LLA_on_Linux
-Source0:   %{name}/%{name}-%{version}.tar.gz
-BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root
-BuildRequires: libmicrohttpd-devel >= 0.4.0, libcppunit-devel, protobuf-devel >= 2.1.0
-Requires:      libmicrohttpd >= 0.4.0, libcppunit, protobuf >= 2.1.0
+Name:           ola
+Version:        0.9.7
```

Comment:
Any reason not to autogenerate this using configure, so it at least tracks the releases, or do you have to do other stuff external to updating this version?